pymysql/cursors.py

```diff
@@ -396,6 +390,7 @@ def fetchall(self):
         Fetch all, as per MySQLdb. Pretty useless for large queries, as
         it is buffered. See fetchall_unbuffered(), if you want an unbuffered
         generator version of this method.
+
         """
```

Comment:
FYI, empty line at end of docstring recommendation was removed.
https://hg.python.org/peps/rev/a7b2b2d5b1d4